Userland/Services/LookupServer/LookupServer.cpp

```diff
@@ -122,6 +136,8 @@ void LookupServer::load_etc_hosts()
         builder.append(".in-addr.arpa"sv);
```

Comment:
`try_append`, and on line above. (I can't seem to select multiple lines using my phone. 😅 )